frontend/src/app/shared/scss/_colors.scss

```diff
@@ -1,32 +1,62 @@
-/* Variables */
-$main-color: #6a983c;
-$main-dark-color: #46760a;
+/* Primary Color */
+$primary-color: #151515;
+$primary-color-010: #575757;
+$primary-color-020: #a9a9a9;
+$primary-color-030: #d1d1d1;
+$primary-color-040: #ebebeb;
+$primary-color-050: #f5f5f5;
+$primary-color-060: #f9f9f9;
+$primary-color-070: #ffffff;
+
+/* Secondary Color */
+$secondary-color: #6a983c;
+$secondary-color-010: #46760a;
+$secondary-color-020: #92c064;
+$secondary-color-030: #c8deb3;
+$secondary-color-030: #f4f8ec;
+
+/* Additional Color - Brown*/
+$additional-color: #d0a866;
+$additional-color-010: #b28a48;
+$additional-color-020: #ecd2a6;
+$additional-color-030: #faedd8;
+$additional-color-030: #fff9f0;
+
+/* Additional Color - Brick*/
+$additional-color: #e5704b;
+$additional-color-010: #c7522d;
+$additional-color-020: #eb8d70;
+$additional-color-030: #f7c6b7;
+$additional-color-030: #fff1ed;
```

Comment:
E frumos ce ai facut, mai ales ca ai si lasat comentariu si stii ce culoare e doar din denumire dar....


urmatoarele greseli.
La fiecare set ( de copy / paste :D ) ai de 2x **-030**  la final , in loc sa ai **-030** , **-040**, dar nu asta e relevant.

Tu suprascrii variabila cu alte valori. 
Gen o sa fie luat in seama doar ultimul set de jos , ultimele valori.

Ce poti sa faci setul asta sa se numeasca
`$error-color`

Reply:
fixed